integration-tests/src/test/resources/jwt_auth/readonlyrest.yml

```diff
@@ -37,7 +37,8 @@ readonlyrest:
     - name: jwt3
       signature_key: "1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890.1234567890"
       user_claim: user
-      roles_claim: roles
+      group_ids_claim: role_ids
```

Comment:
more realistic example would be claim like this:
```
"roles": [
  { "id":"r1", "name": "role 1"},
  { "id":"r2", "name": "role 2"}
]
```

are we able to modify the test suite?